docs/build/quickstarts.md

```diff
@@ -1,3 +1,200 @@
+
+### How deals are matched
+
+In order for a Deal to be deployed, there should be Providers willing to host it. That process is governed by the `Matcher` smart contract.
+
+It's important that Matcher makes it possible for a single Deal to be deployed on several hosts to provide High Availability.
+
+A Deal specifies different parameters, like `price per epoch`, `minimum collateral`, `effectors` and `target number of workers`.
+
+These parameters are matched against Providers' Market Offers, which specify `minimum price per epoch`, `maximum collateral` and `available effectors`.
+
+You can read more about effectors [HERE](https://www.youtube.com/watch?v=tgTUtfb0Ok8).
+
+Providers register their Market Offers on chain in a contract called `Matcher`, and each `deal deploy` calls that contract to find Offers compatible with that Deal. It should match the price, collateral and available `effectors`.
+
+Each match should find at least `target number of workers` Providers whose offers are compatible with a given Deal.
+
+After match happened, Providers recieve an event from Chain that commands them to deploy Deal to their chosen Compute Peer.
+
+You can read a more detailed description of Matching process [HERE](https://www.youtube.com/watch?v=tgTUtfb0Ok8).
+
+## What are workers
+
+When Deal is matched against Market Offers, it gets deployed to a number of Compute Peers.
+
+For every deal, a Compute Peer creates a secure samespace to isolate functions of that Deal from other Deals. That secure namespace is called a Worker. After succesful deployment, a Worker holds all Services and Spells.
+
+Since Workers are isolated from each other, Service and Spells names and aliases will never have a collision with Services and Spells in other Workers.
+
+In order to reach these Services and Spells and call functions on them, it's required to know two things: Compute Peer ID and Worker ID.
+
+For purposes of this document, it's enough to know that a Worker is what holds a set of Services and Spells for a single Deal.
+
+## What are subnets
+
+Subnet is a set of Workers each holding the same Deal.
+
+It's important to diversify where Deal is hosted to provide High Availability. So Workers of a single Deal will always be on different Compute Peers, and most likely these Compute Peers will come from different Providers.
+
+So, Subnet unites several Compute Peers of several Providers. A single Subnet corresponds to a single Deal, so each Compute Peer of a Subnet hosts a Worker which contains all the Services and Spells defined by the Deal's `AppCID`.
+
+Information about Subnet participants is stored on Chain. So it's a public information that's easy to retrieve from Chain knowing a Deal ID. Clients use that information to resolve Subnets to a list of Workers, and access Services and Spells inside these Workers.
```

Comment:
seems a bit superfluous given that the deal discussion leading up to this.
if you want to drive home the lookup .. how about a one liner instead:
Since deals are on-chain and therefore public, it is easy to resolve Subnets via the deal's specified workers which in turn facilitates access to each worker's services and spells.

Reply:
Not sure I see a big difference here :)